Dockerfile

```
@@ -0,0 +1,30 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+# install kubectl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && \
```

Prefer `$()` to backticks
```suggestion
RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
```